usaspending_api/references/urls.py

```diff
@@ -15,7 +15,9 @@
     url(r'^locations/geocomplete', views.LocationGeoCompleteEndpoint.as_view()),
     url(r'^agency/$', views.AgencyEndpoint.as_view(mode_list)),
     url(r'^agency/autocomplete', views.AgencyAutocomplete.as_view()),
-    url(r'^recipients/autocomplete', views.RecipientAutocomplete.as_view()),
+    url(r'^recipients/$', views.RecipientViewSet.as_view(mode_list), name='recipient-list'),
```

Comment:
While we're in here making changes, what do you think about moving away from the `references/` prefix in our endpoints? Technically doing this now would be jumping the gun, since we haven't laid down our new API guiding principles yet, but I've long thought the `references` prefix doesn't have any meaning, since objects like recipients, cfda, and the like are full-fledged members of the fedspending data ecosystem.

Not something we need to change right now, but just planting the seed...